projects/packages/forms/package.json

```
@@ -17,26 +17,63 @@
 	"scripts": {
 		"build": "pnpm run clean && pnpm run build:contact-form && pnpm run build:dashboard",
```

You could run the block build here too:

```suggestion
		"build": "pnpm run clean && pnpm run build:blocks && pnpm run build:contact-form && pnpm run build:dashboard",
```